src/marker_group.js

```diff
@@ -59,12 +64,12 @@ class MarkerGroup {
      * @param {any} html
      * @param {Marker} markerLayer
      * @param {EditSession} session
-     * @param {{ firstRow: any; lastRow: any; }} config
+     * @param {LayerConfig} config
      */
     update(html, markerLayer, session, config) {
         if (!this.markers || !this.markers.length)
             return;
-        var visibleRangeStartRow = config.firstRow, visibleRangeEndRow = config.lastRow;
+        var visibleRangeStartRow = config.firstRow, visibleRangeEndRow = 100;
```

Comment:
should this be changed?

Reply:
ah whoops, that was just a change I made while playing around thanks for catching that 😅

Reply:
Removed that change